### PR TITLE
Fix parsing of Lint error message on windows

### PR DESCRIPTION
**Pull request by samtkaplan**
On windows the filePath includes `C:` which breaks the current `split(":")` logic which depends on the error message being in the second element of `splittedline`.  This simply removes the filename from `line` before the call to `split`, and then assumes that the error message is in the first element of `splittedline`.  This seems to work on windows.  I have not tested on mac or linux.

### CODE REVIEW

File: lib/linter-julia.coffee [hunk 1]
```diff
@@ -23,9 +23,9 @@ doSomeMagic = (data,textEditor) ->
   lines = data.split("\n")
   for line in lines
     try
-      splittedline = line.split(":")
-      numbers = splittedline[1].split(" ")
-      if numbers[1] in ignore
+      splittedline = line.replace(filePath+":","").split(":")
+      numbers = splittedline[0].split(" ")
+      if numbers.length < 2 || numbers[1] in ignore
```
Comment (TeroFrondelius) [inline lib/linter-julia.coffee]: Can you explain, why this is needed: `numbers.length < 2 ||`? I'm not against it, just curious.

Comment (TeroFrondelius) [inline lib/linter-julia.coffee]: Why is this needed: `numbers.length < 2 ||`? I'm not against it just curious.

Comment (samtkaplan) [inline lib/linter-julia.coffee]: I was hitting a problem where `numbers` was an array with only one element.  To tell the truth, I'm not sure why this was happening.  At the time I made the change, I thought that it was because I was shrinking the string before doing the split, but, actually, I'm not sure if that makes sense.  Anyways, thanks for merging, and glad to hear there is another solution in the works.
